rules/rules.py

```
@@ -0,0 +1,243 @@
+
+        while True:
+            try:
+                reaction, user = await self.bot.wait_for('reaction_add', check=check, timeout=60.0)
+                role = ctx.guild.get_role(acceptance_role_id)
+                if role:
+                    try:
+                        await user.add_roles(role)
+                        try:
+                            await user.send("Thank you for accepting the rules. Please remember that failing to follow them will result in moderation.")
+                        except discord.Forbidden:
+                            pass  # If we can't DM the user, just pass silently
```

Ruff (0.8.2)

Use `contextlib.suppress(discord.Forbidden)` instead of `try`-`except`-`pass`

(SIM105)